lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -55,12 +55,11 @@ export default ({ configDir, configType, entries, dll, outputDir, cache, babelOp
         }),
         template: require.resolve(`../templates/index.ejs`),
       }),
-      new webpack.DefinePlugin({ 'process.env': stringified }),
       new CaseSensitivePathsPlugin(),
       new Dotenv({ silent: true }),
       // graphql sources check process variable
       new DefinePlugin({
-        process: JSON.stringify(true),
+        process: { browser: true, env: stringified },
         NODE_ENV: JSON.stringify(process.env.NODE_ENV),
```

Comment:
is the above NODE_ENV necessary or is it a bug by the way ?
leaving as it is for now.

Reply:
It's intentional, from https://github.com/storybooks/storybook/pull/6215

Reply:
ok